src/components/Book.js

```diff
@@ -3,43 +3,58 @@ import './App.css';
 import { useSelector, useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
 import AddBook from './AddBook';
+import Endpoints from '../Endpoints';
 
 function Book() {
   const books = useSelector((state) => state.booksReducer);
   const dispatch = useDispatch();
   const removeFromStore = (id) => {
-    dispatch(removeBook(id));
+    fetch(`${Endpoints.baseUrl}/${id}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        item_id: id,
+      }),
+    }).then((res) => res.text()).then((data) => {
+      console.log(data);
+      dispatch(removeBook(id));
+    }).catch((error) => console.log(error));
   };
```

Comment:
- [ ] Kindly remove `console.log` statement. It may expose critical data to the user and compromise security.

Reply:
Thanks, done!